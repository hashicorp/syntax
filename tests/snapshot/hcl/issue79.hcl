block {
    attr = var.test
    attr2 = var.test == "foo" ? "true" : "false"
    nested_obj = {
        attr = var.test
        attr2 = var.test == "foo" ? "true" : "false"
    }
}