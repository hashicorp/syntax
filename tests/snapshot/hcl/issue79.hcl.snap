>block {
#^^^^^ source.hcl meta.block.hcl entity.name.type.hcl
#     ^ source.hcl meta.block.hcl
#      ^ source.hcl meta.block.hcl punctuation.section.block.begin.hcl
>    attr = var.test
#^^^^ source.hcl meta.block.hcl
#    ^^^^ source.hcl meta.block.hcl variable.declaration.hcl variable.other.readwrite.hcl
#        ^ source.hcl meta.block.hcl variable.declaration.hcl
#         ^ source.hcl meta.block.hcl variable.declaration.hcl keyword.operator.assignment.hcl
#          ^ source.hcl meta.block.hcl variable.declaration.hcl
#           ^^^ source.hcl meta.block.hcl
#              ^ source.hcl meta.block.hcl keyword.operator.accessor.hcl
#               ^^^^ source.hcl meta.block.hcl variable.other.member.hcl
>    attr2 = var.test == "foo" ? "true" : "false"
#^^^^ source.hcl meta.block.hcl
#    ^^^^^ source.hcl meta.block.hcl variable.declaration.hcl variable.other.readwrite.hcl
#         ^ source.hcl meta.block.hcl variable.declaration.hcl
#          ^ source.hcl meta.block.hcl variable.declaration.hcl keyword.operator.assignment.hcl
#           ^ source.hcl meta.block.hcl variable.declaration.hcl
#            ^^^ source.hcl meta.block.hcl
#               ^ source.hcl meta.block.hcl keyword.operator.accessor.hcl
#                ^^^^ source.hcl meta.block.hcl variable.other.member.hcl
#                    ^ source.hcl meta.block.hcl
#                     ^^ source.hcl meta.block.hcl keyword.operator.hcl
#                       ^ source.hcl meta.block.hcl
#                        ^ source.hcl meta.block.hcl string.quoted.double.hcl punctuation.definition.string.begin.hcl
#                         ^^^ source.hcl meta.block.hcl string.quoted.double.hcl
#                            ^ source.hcl meta.block.hcl string.quoted.double.hcl punctuation.definition.string.end.hcl
#                             ^ source.hcl meta.block.hcl
#                              ^ source.hcl meta.block.hcl keyword.operator.hcl
#                               ^ source.hcl meta.block.hcl
#                                ^ source.hcl meta.block.hcl string.quoted.double.hcl punctuation.definition.string.begin.hcl
#                                 ^^^^ source.hcl meta.block.hcl string.quoted.double.hcl
#                                     ^ source.hcl meta.block.hcl string.quoted.double.hcl punctuation.definition.string.end.hcl
#                                      ^ source.hcl meta.block.hcl
#                                       ^ source.hcl meta.block.hcl keyword.operator.hcl
#                                        ^ source.hcl meta.block.hcl
#                                         ^ source.hcl meta.block.hcl string.quoted.double.hcl punctuation.definition.string.begin.hcl
#                                          ^^^^^ source.hcl meta.block.hcl string.quoted.double.hcl
#                                               ^ source.hcl meta.block.hcl string.quoted.double.hcl punctuation.definition.string.end.hcl
>    nested_obj = {
#^^^^ source.hcl meta.block.hcl
#    ^^^^^^^^^^ source.hcl meta.block.hcl variable.declaration.hcl variable.other.readwrite.hcl
#              ^ source.hcl meta.block.hcl variable.declaration.hcl
#               ^ source.hcl meta.block.hcl variable.declaration.hcl keyword.operator.assignment.hcl
#                ^ source.hcl meta.block.hcl variable.declaration.hcl
#                 ^ source.hcl meta.block.hcl meta.braces.hcl punctuation.section.braces.begin.hcl
>        attr = var.test
#^^^^^^^^ source.hcl meta.block.hcl meta.braces.hcl
#        ^^^^ source.hcl meta.block.hcl meta.braces.hcl meta.mapping.key.hcl variable.other.readwrite.hcl
#            ^ source.hcl meta.block.hcl meta.braces.hcl
#             ^ source.hcl meta.block.hcl meta.braces.hcl keyword.operator.assignment.hcl
#              ^ source.hcl meta.block.hcl meta.braces.hcl
#               ^^^^^^^^^ source.hcl meta.block.hcl meta.braces.hcl
>        attr2 = var.test == "foo" ? "true" : "false"
#^^^^^^^^ source.hcl meta.block.hcl meta.braces.hcl
#        ^^^^^ source.hcl meta.block.hcl meta.braces.hcl meta.mapping.key.hcl variable.other.readwrite.hcl
#             ^ source.hcl meta.block.hcl meta.braces.hcl
#              ^ source.hcl meta.block.hcl meta.braces.hcl keyword.operator.assignment.hcl
#               ^ source.hcl meta.block.hcl meta.braces.hcl
#                ^^^^^^^^^ source.hcl meta.block.hcl meta.braces.hcl
#                         ^^ source.hcl meta.block.hcl meta.braces.hcl keyword.operator.hcl
#                           ^ source.hcl meta.block.hcl meta.braces.hcl
#                            ^ source.hcl meta.block.hcl meta.braces.hcl string.quoted.double.hcl punctuation.definition.string.begin.hcl
#                             ^^^ source.hcl meta.block.hcl meta.braces.hcl string.quoted.double.hcl
#                                ^ source.hcl meta.block.hcl meta.braces.hcl string.quoted.double.hcl punctuation.definition.string.end.hcl
#                                 ^ source.hcl meta.block.hcl meta.braces.hcl
#                                  ^ source.hcl meta.block.hcl meta.braces.hcl keyword.operator.hcl
#                                   ^ source.hcl meta.block.hcl meta.braces.hcl
#                                    ^ source.hcl meta.block.hcl meta.braces.hcl string.quoted.double.hcl punctuation.definition.string.begin.hcl
#                                     ^^^^ source.hcl meta.block.hcl meta.braces.hcl string.quoted.double.hcl
#                                         ^ source.hcl meta.block.hcl meta.braces.hcl string.quoted.double.hcl punctuation.definition.string.end.hcl
#                                          ^ source.hcl meta.block.hcl meta.braces.hcl
#                                           ^ source.hcl meta.block.hcl meta.braces.hcl keyword.operator.hcl
#                                            ^ source.hcl meta.block.hcl meta.braces.hcl
#                                             ^ source.hcl meta.block.hcl meta.braces.hcl string.quoted.double.hcl punctuation.definition.string.begin.hcl
#                                              ^^^^^ source.hcl meta.block.hcl meta.braces.hcl string.quoted.double.hcl
#                                                   ^ source.hcl meta.block.hcl meta.braces.hcl string.quoted.double.hcl punctuation.definition.string.end.hcl
>    }
#^^^^ source.hcl meta.block.hcl meta.braces.hcl
#    ^ source.hcl meta.block.hcl meta.braces.hcl punctuation.section.braces.end.hcl
>}
#^ source.hcl meta.block.hcl punctuation.section.block.end.hcl